src/app.ts

```diff
@@ -45,10 +55,30 @@ async function app() {
 
   logger.info(`Running SUSHI ${getVersion()}`);
 
-  // Check that input folder is specified
+  // If no input folder is specified, set default to current directory
   if (!input) {
-    logger.error('Missing path to FSH definition folder.');
-    program.help();
+    input = path.join('.');
```

Comment:
I think `path.join('.')` is basically a no-op, returning back `'.'`.

Reply:
I think you're right. The documentation says:

> Zero-length path segments are ignored. If the joined path string is a zero-length string then '.' will be returned, representing the current working directory.

Do you think it is safer to just assign `'.'` or to do `path.join()`? Is there ever a case where current directory would have anything else as a part of it?

Reply:
I think just assigning `'.'` should be fine as I'm 99.99% sure that's what `path.join('.')` will return every time.  If you're looking for it to return the absolute path of the working directory, I think you need to do that a different way.

Reply:
It doesn't need the relative path. It also seems like `path.join()` with no arguments passed in also returns `'.'`, but I'll just assign the string.